pgdog/src/admin/healthcheck.rs

```diff
@@ -0,0 +1,43 @@
+use super::prelude::*;
+use crate::backend::{databases::databases, pool::monitor::Monitor};
+
+#[derive(Default)]
+pub struct Healthcheck {
+    id: Option<u64>,
+}
+
+#[async_trait]
+impl Command for Healthcheck {
+    fn name(&self) -> String {
+        "HEALTHCHECK".into()
+    }
+
+    fn parse(sql: &str) -> Result<Self, Error> {
+        let parts = sql.split(" ").collect::<Vec<_>>();
```

Comment:
might want a more liberal whitespace here, in case of newline etc

Reply:
Agreed. We should write our own parser for these, the right way (regex :D)